sdk/communication/tests.yml

```diff
@@ -23,11 +23,10 @@ stages:
         ServiceDirectory: communication
         DeployArmTemplate: true
         TestSamples: true
-        EnvVars:
-          AZURE_TEST_RUN_LIVE: 'true'
```

Comment:
I think we need this, right?

Reply:
This has been put into the python subconfig